Vite related docs and changelog update (#1037)
wasp-lang/wasp: web/docs/getting-started.md

## 1. Requirements

> You need to have `node` (and `npm`) installed on your machine and available in `PATH`. We rely on the latest Node.js LTS version, and currently, that's version `v18.14.0`.

Reviewer: ```suggestion
You must have `node` (and `npm`) installed on your machine and available in `PATH`. We rely on the latest Node.js LTS version (currently `v18.14.0`).
```

You need to have `node` (and `npm`) installed on your machine and available in `PATH`. We rely on the latest Node.js LTS version, and currently, that's version `v18.14.0`.

You can check the `node` version by running:

> ```shell-session

Author: I replaced `shell-session` with `shell` since the highlighting works better with `shell`
